models/alkhalaf2024/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+title: Ontology to Explain SARS-CoV-2 Variants' Effects
+acronym: OntoEffect
+issued: 2023
+modified: 
+contributor:
+ - https://dblp.org/pid/292/3745
+ - https://dblp.org/pid/25/929-2
+ - https://dblp.org/pid/201/2998
+keyword:
+ - genetic mutations
+ - viral fitness
+ - host-virus interaction
+theme: Class Q - Science 
+editorialNote: 
+ontologyType:
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+ - interoperability
+ - ontological analysis
+context:
+ - research
+source:
+ - https://tinyurl.com/5n8c7k8t
```

Comment:
We should not use a GoogleDrive link with some images as the source. It should, instead, be the paper which includes these figures. 

http://doi.org/10.5220/0012183900003598

Reply:
O DOI que você adicionou não resolve para o artigo, mas para o proceedings que inclui o artigo.

Reply:
Thanks for noticing! Fixed.